tests/testinator/needs.json

```diff
@@ -0,0 +1,12 @@
+{
+	"libraries": {
+		"testinator": {
+			"repository": "git@github.com:elbeno/testinator.git",
+			"commit": "b6183c43718ccefb7896c47a1d4a9d89388890e7",
+			"project" : {
+				"header-only" : true,
+				"header-directory" : "src/include"
```

Comment:
we should probably add `os.path.join(self.source_directory(...), 'include')` to the paths that we look in. i think this is common enough to warrant it